projects/angular/src/data/datagrid/_datagrid.clarity.scss

```
@@ -957,18 +1022,74 @@
     .datagrid-footer {
       min-height: tables-variables.$clr-table-compact-row-height;
```

Check with designers and figma.

```suggestion
      min-height: 28px;
```